tests/other_test.go

```diff
@@ -23,6 +23,10 @@ func TestRunBasicExample(t *testing.T) {
 		ResourceGroup:      resourceGroup,
 		BestRegionYAMLPath: regionSelectionPath,
 		CloudInfoService:   sharedInfoSvc,
+
+		TerraformVars: map[string]interface{}{
+			"elasticsearch_version": "8.10", // Always lock this test into the latest supported elasticsearch version
```

Comment:
should this not be testing 8.7? We already have a test for 8.10 in pr_test.go

Reply:
ah hmm, looks like `TestRunCompleteExampleOtherVersion` is running 8.7. So this is probably fine